.github/workflows/main.yml

```diff
@@ -16,7 +16,10 @@ jobs:
       with:
         go-version: '1.17' # The Go version to download (if necessary) and use.
     - run: sudo apt install -y universal-ctags pandoc
-    - run: go test ./...
+    - run: mkdir .tools && wget https://github.com/daedaleanai/llvm-project/releases/download/ddln-llvm14-rc/llvm-14.0.0.tar.gz && tar -xf llvm-14.0.0.tar.gz -C .tools
```

Comment:
Github workflows has weird free quota and pricing policy if free quota exceeded. Let's check that downloading gigabytes of data will not cost a fortune

Reply:
What I read is that the pricing policy only applies for private repositories, which this is not, so I think this should be fine. 

> GitHub Actions usage is free for both public repositories and self-hosted runners. For private repositories, each GitHub account receives a certain amount of free minutes and storage, depending on the product used with the account. Any usage beyond the included amounts is controlled by spending limits.

Also it seems that by default there is an option to set limits to the expenses (and for monthly billed customers the default is zero):
> If you are a monthly-billed customer, your account will have a default spending limit of 0 US dollars (USD), which prevents additional usage of minutes or storage for private repositories beyond the amounts included with your account. If you pay your account by invoice, your account will have an unlimited default spending limit. For more information, see "[About spending limits](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions#about-spending-limits)."

Lastly, they seem to charge based on minutes of execution and storage. I didn't read anything about network bandwidth or usage.

https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions

So I think this should be fine

Reply:
Sent an internal email to check the policy, in the meantime, I think we can go ahead with this.